CMakeLists.txt

```diff
@@ -441,32 +441,6 @@ include_directories(SYSTEM ${TreeSitter_INCLUDE_DIRS})
 
```

Comment:
The find_package() call needs to specify the minimum required version then, right?

Reply:
tree-sitter does not really set a version (or a meaningless) in their `pckconfig`: https://github.com/tree-sitter/tree-sitter/blob/c669e5ee159e0c59a3f094327a01dd688bc67c56/Makefile#L1 (this version didn't change since ages). This is why I didn't set a version requirement

Reply:
we can also just leave the try_build commands there for longer

Reply:
Then either the PR isn't needed or should be changed to tell the user they're using an unsupported version. In the latter case, the ifdefs can be removed but the CMake checks need to be kept and turned into a fatal error with a descriptive message.